package.json

```diff
@@ -29,15 +29,18 @@
     "format": "prettier --write \"{src,tests}/**/*.ts\"",
     "check-format": "prettier --list-different \"{src,tests}/**/*.ts\""
   },
+  "dependencies": {
+    "@typescript-eslint/experimental-utils": "4.28.0"
+  },
   "peerDependencies": {
-    "eslint": "^5.0.0 || ^6.0.0 || ^7.0.0"
+    "eslint": "^5.0.0 || ^6.0.0 || ^7.0.0",
+    "typescript": ">=3.3.1"
```

Comment:
I'm not sure that adding `typescript` as a peer dependency is a good idea - in some cases, it could cause problems. For example, I don't use rules that require it, have another version of `typescript` as a dependency, and `npm@7` will try to install it and die because of incompatible `typescript` versions.

Reply:
thanks for letting us know about it, I was also not sure about it, but the thing is that transitive dependency of `@typescript-eslint/experimental-utils` (`tsutils`) already puts it as a peer dependency. So the problem will be still there.
Do you have some better idea? 

We need `@typescript-eslint/experimental-utils` as it's used in `.d.ts` files. It's also used a bit in `.js` but that's probably could be cleaned out.

Reply:
In this case, I don't know. Maybe make sense to add it as a direct dependency.